src/scope.js

```diff
@@ -400,19 +422,19 @@ class Scope {
     }
 
     __isClosed() {
-        return this.__left === null;
+        return this.__left === null || this.__left === undefined;
```

Comment:
They had some parts of the code that had != null and == null and I changed those to triple equals with undefined. I can change this one back.